src/main/kotlin/com/exactpro/th2/conn/amqp/MessagePublisher.kt

```diff
@@ -50,12 +50,12 @@ class MessagePublisher(
     private val counters: Map<Direction, Counter.Child> = EnumMap<Direction, Counter.Child>(Direction::class.java).apply {
         // FIXME: use DEFAULT_SESSION_ALIAS_LABEL_NAME variable
         put(Direction.FIRST, Counter.build().apply {
-            name("th2_conn_incoming_msg_quantity")
+            name("th2_conn_incoming_msg_quantity_$sessionAlias")
```

Comment:
Redundant change because session_alias is already included into labels.
Please read more about labels by link https://prometheus.io/docs/practices/instrumentation/#use-labels